src/main/java/org/launchcode/techjobs/persistent/models/Job.java

```diff
@@ -3,42 +3,42 @@
 import javax.persistence.*;
 
 @Entity
-public class Job{
+public class Job extends AbstractEntity{
 
-    @Id
-    @GeneratedValue
-    private int id;
+//    @Id
+//    @GeneratedValue
+//    private int id;
 
-    private String name;
+//    private String name;
 
-    private String employer;
+    private String Employer;
     private String skills;
 
     public Job() {
     }
 
     public Job(String anEmployer, String someSkills) {
         super();
-        this.employer = anEmployer;
+        this.Employer = anEmployer;
```

Comment:
```suggestion
        this.employer = anEmployer;
```
I would change this back to `employer` to prevent it from being mistaken for the Employer class name vs. the variable/property name.